html/changelogs/unfit_shuttles.yml

```diff
@@ -0,0 +1,41 @@
+################################
+# Example Changelog File
+#
+# Note: This file, and files beginning with ".", and files that don't end in ".yml" will not be read.  If you change this file, you will look really dumb.
+#
+# Your changelog will be merged with a master changelog. (New stuff added only, and only on the date entry for the day it was merged.)
+# When it is, any changes listed below will disappear.
+#
+# Valid Prefixes: 
+#   bugfix
+#   wip (For works in progress)
+#   tweak
+#   soundadd
+#   sounddel
+#   rscdel (general adding of nice things)
+#   rscadd (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   spellcheck (typo fixes)
+#   experiment
+#   tgs (TG-ported fixes?)
+#################################
+
+# Your name.  
+author: Unid
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+changes: 
+  - rscadd: The following shuttles have been moved to a new shuttle system which doesn't use areas: mining shuttle, research shuttle, syndicate shuttle, emergency shuttle, vox shuttle, salvage shuttle
+  - rscdel: Areas of the shuttles above no longer clutter the map, and are replaced by invisible docking ports which do not affect gameplay in any way
+  - rscdel: Mining and research shuttle computers no longer work (and will soon be removed)
+  - rscadd: Added a new shuttle control console which can connect to any shuttle which allows it (by default - mining shuttle, research shuttle, salvage shuttle with password)
+  - experiment: Added dynamic lighting and lights to all shuttles
+  - experiment: Shuttles using the new shuttle system now throw all unanchored objects back when moving
```

Comment:
It happens when they move (be it to another location or to/from transit)

Note that after testing it, I didn't really like its behaviour on syndicate/vox/salvage shuttles, which have tons of weapons and equipment lying around on the tables so the whole thing becomes a huge mess, so I might only enable it for mining, research and custom shuttles